demo/app/main-view-model.ts

```diff
@@ -48,27 +48,38 @@ export class HelloWorldModel extends Observable {
     };
 
     setTimeout(() => {
-      console.log('setupCallListener!');
+      console.log('Registering call listeners');
       setupCallListener(callListener);
-    }, 10000)
+    }, 15000)
 
-    // listener for push notifications (incoming calls)
-    const pushListener = {
-      onPushRegistered(accessToken, deviceToken) {
-        dialogs.alert('push registration succeded');
-      },
-      onPushRegisterFailure (error) {
-        dialogs.alert(`push registration failed: ${error}`);
-      }
-    };
+  
 
-    setupPushListener(pushListener);
+    setTimeout(() => {
+        // listener for push notifications (incoming calls)
```

Comment:
Nit: I think the comment line is misaligned here. Maybe is because you are mixing tabs and spaces? If so, I would prefer only having spaces in the code.

Reply:
Hi manuel agreed, ill start using spaces from now on. fixed this specific misalignment 👍